### ISSUE TITLE
When to call scheduler.step()

### ISSUE THREAD

**Issue by ChewKinWhye** [tourdeml/abel-pytorch#2]
In the paper, it states that "we keep track of the changes in weight norm between subsequent epochs". However, in the usage section, scheduler.step() is called after every batch.

Could I clarify if scheduler.step() should be called after every batch or after every epoch?

Thanks!
